### ISSUE TITLE
Modpack files optimization and clean up!

### ISSUE THREAD

**Issue by InFuRiTy** [LunaPixelStudios/Fear-Nightfall#144]
I'm guessing the pack was released in a hurry for the Halloween hype and you guys forgot to tidy up both the client and server packs for release.

Server pack has many Client mods that do nothing and a RIDICULOUS amount of config files with most of them pointing to mods that don't exist in the pack. That's an issue for servers/hosts that run the pack from a Pterodactyl panel because it has a 250 files limit per directory so it ends up not showing everything in case you need to tweak something and it's not included in the list. That's our case sadly with our community server. I had to manually delete most of those files in order to get all the files to show up. The server still operates fine!

Client pack also has that problem but i'm guessing it's not an issue since PCs don't have directory limits or a problem running server mods. (since SP and lan are considered local server environments)

Also i dunno it was a download error or corruption of some sort but both times we setup our server the time control config was reset  to 10 min day/night and we had to re-configure it manually.

Awesome pack nonetheless. We really love it!! Keep up the good work!!

**Comment by nvb-uy** [comment]
There are some residues from the base of the modpack because initially it was based on Medieval MC, I'll be cutting off the residues, but do keep in mind that this is still not really an issue. I've never saw pterodactyl have issues due to a lot of config files, and my test server is running on pterodactyl with no issues. This might be a limitation put by your host provider.

For reference, Fear Nightfall has 350 config files (and folders), while Prominence II has 410. You're the first person to ever report an issue from this happening within Fear Nightfall, and I've never seen anyone have a similar issue with Prominence which has considerably a lot more files. (On my pterodactyl based server running with duck panel there's absolutely no issues displaying files on the configs or mods folders)

The reason why your time control config wasn't set in your server is because you didn't add the defaultconfigs folder, please follow the update guide for servers on how to update it: https://youtu.be/9dLMOiPp_Vk

If you were told that the directories/files are an issue by your server host, then I'd suggest you to switch to another host, as this is NOT an issue that should happen normally.

Something to keep in mind is that even if you try to cut off files, most of them are still necessary, and besides that, removing configs isn't really an optimization as it has no impact whatsoever in your server's performance besides your panel taking a bit more to fetch all files. This is sometimes inevitable, because while Fear Nightfall might have a few configs from removed mods that were in MMC, other modpacks such as Prominence have the files required for the modpack which end up being hundreds, and you cannot really cut them without messing up with the modpack, configs are an essential part of modpacks.

TLDR; While there are residues, It's not optimization. Configs are also very necessary, and some packs have over 400 files natively because they're required. Your issue is likely imposed by your host provider, or if you're self hosting your pterodactyl instance you might be able to change that limit.
